Source/Painting/SvgLinearGradientServer.Drawing.cs

```diff
@@ -147,6 +147,29 @@ private GradientPoints ExpandGradient(ISvgBoundable boundable, PointF specifiedS
             return new GradientPoints(effectiveStart, effectiveEnd);
         }
 
+        class MyPointFEqualityComparer : EqualityComparer<PointF>
```

Comment:
We cannot infer processing from this class name.
Please name it appropriately.

Reply:
Sorry, I didn't write this code but just pasted it from https://github.com/svg-net/SVG/issues/1008#issue-1370121240
Actually it is exactly what the class name suggests: It compares two PointF objects for equality. How else would you name this class? Just PointFEqualityComparer?

Reply:
I agree that `PointFEqualityComparer` would be an adequate name, just omit the "My".